### ISSUE TITLE
Sorry for asking but I could use some help picking which pool to mine from. 

### ISSUE THREAD

**Issue by cmbconcretefire** [MultiPoolMiner/MultiPoolMiner#927]
Ps, I am still looking into issue #866 & #903. I think I can get it working. 

You need to know my rig power. 2 dedicated rigs. Nothing else running on them besides miners. 
1 rig = 6 x 1060 (6gb) , the other rig = 6 x 1070 (8gb) (not Ti's) 

I used to mine at MPH exclusively until I read that Zpool may be better because of all the switching done at MPH along with auto exchange taking a bite for exchange fees. So, I switched to Zpool exclusively for the past month. I seem to be getting a steady payout of about 0.003 btc every 24 hrs / give or take. 

I just noticed one of the miners was mining to Ahashpool even though that pool IS NOT in my bat file. I plugged in my wallet info at Ahashpool, and sure enough, I have a balance there of 0.001. 

The bottom line is this: > I have no clue how to "pick" what pool I should be mining at for max profitability given the equipment that I have. Any words of wisdom appreciated. MPH? Ahashpool? Hashrefinery? Any of the others listed in the "pools" folder?

**Comment by UselessGuru** [comment]
>  Any words of wisdom appreciated. MPH? Ahashpool? Hashrefinery? Any of the others listed in the "pools" folder?

None can give you THE correct answer. Do not use them all at the same time, this can cause pool hopping and will reduce profit.
